scripts/python-version-check.py

```diff
@@ -16,7 +16,7 @@
 from sys import executable, exit, version_info
 
 PYTHON_VER_MIN = (3, 9)
-PYTHON_VER_MAX = (3, 10)
+PYTHON_VER_MAX = (3, 11)
```

Comment:
Should we update `/docs/Development.md `line 147?

"After cloning the repo, you'll need a python version that's >= 3.9 **but < 3.11**."

Reply:
updated.. thanks